src/test/regress/Pipfile

```diff
@@ -5,7 +5,7 @@ verify_ssl = true
 
 [packages]
 mitmproxy = {editable = true, ref = "main", git = "https://github.com/citusdata/mitmproxy.git"}
-construct = "==2.9.45"
+construct = "*"
```

Comment:
I don't exactly know what I ran here, but it is odd to me that it changed it to a `*`. Will have a look once I get back to see if I can pin this to the current version to prevent unexpected changes in the future

Reply:
I did not really understand how it does not cause any problem for us to have many `*` versions in the rest of this file. Maybe there is not much risk at all.

Reply:
could be, given there are more it might be ok?